tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "target": "ES2015",
+    "module": "commonjs",
+    "moduleResolution": "node"
+  },
```

Comment:
For your fork, I would recommend to enable [strict validations](https://www.typescriptlang.org/tsconfig#strict) to allow fewer ways to trick or work around the type system.